### ISSUE TITLE
Initial data load into main workspace

### ISSUE THREAD

**Issue by samhowes** [samhowes/foothills-data#19]

**Comment by samhowes** [comment]
@revsean Completed initial upload into the new adult and child workspaces. Please review. 

Lets create new issues for individual data items discovered. Already, I see:

- [ ] #29 
- [ ] #30

**Comment by samhowes** [comment]
I'm also seeing a lot of "Joined Group" Activities, i.e 69 people Joined a group on 9/24: is this how you entered the data? Or do you suspect there is a bug?

**Comment by revsean** [comment]
> I'm also seeing a lot of "Joined Group" Activities, i.e 69 people Joined a group on 9/24 - 

This is correct. We created some groups on that day and added a bunch of people.

**Comment by revsean** [comment]
New Issue 

- [ ] [Incorrect Activity Types for Worship Data](https://github.com/samhowes/foothills-data/issues/32)
